unittests/files/opt/seagate/sspl/setup/resource_map/test_storage.py

```diff
@@ -130,6 +130,86 @@
     """
 ENCLOSURE_RESPONSE_EMPTY = {}
 
+FANMODULES_RESPONSE = """
+{
+    "status_code": 200,
+    "api-response":{
+        "fan-modules":[
+            {
+                "object-name":"fan-modules",
+                "meta":"/meta/fan-modules",
```

Comment:
Shriya addressed keeping api response alone in a separate file for all FRUs.
After her PR merge #426, please move this response in that file.

Reply:
Done